colcon_bundle/verb/bundle.py

```diff
@@ -159,6 +160,11 @@ def _manage_dependencies(self, context,
 
         installers = self._setup_installers(context)
 
+        print('Checking if dependency has changed since last time...')
```

Comment:
We should print to the logger too `logger.info(msg)`

Reply:
Minor string change: `Checking if dependencies have changed since last bundle...`